packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -73,14 +73,14 @@ class TabPanel extends React.Component {
 
     return (
       <ThemeProvider theme={theme}>
-        <div className={className}>
+        <div className="es-tab-panel">
           <AriaAnnouncer id="announcer" aria-live="assertive">{`${
             this.state.simpleName
           } Sub text is now showing`}</AriaAnnouncer>
-          <TabWrapper className="tab-wrapper">
-            <TabFormatter className="tab-formatter">{elements}</TabFormatter>
+          <TabWrapper className="es-tab-wrapper">
```

Comment:
If we want to go so far as to adding classes to all the subelements, I'd like to use the BEM naming convention in some form. http://getbem.com/naming/
https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/

So es-tab-panel would be the root, and this would be es-tab-panel__wrapper, for example.